packages/playground/website/playwright/deploy_test_setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Store the current branch name
+CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+git checkout 4cbd5ac28
+PLAYGROUND_URL=http://localhost:8000 npx nx run playground-website:build:wasm-wordpress-net
+rm -rf dist/packages/playground/wasm-wordpress-net-old
+mv dist/packages/playground/wasm-wordpress-net dist/packages/playground/wasm-wordpress-net-old
+
+git checkout d7b1a7274
+PLAYGROUND_URL=http://localhost:8000 npx nx run playground-website:build:wasm-wordpress-net
+rm -rf dist/packages/playground/wasm-wordpress-net-new
+mv dist/packages/playground/wasm-wordpress-net dist/packages/playground/wasm-wordpress-net-new
+
+# Checkout back to the original branch
+git checkout $CURRENT_BRANCH
```

Comment:
This script doesn't work for me, but I was able to use the code to generate local folders with it.

My suggestion is that we add an old version as a E2E fixture, similarly [to this Gutenberg file](https://github.com/WordPress/wordpress-playground/blob/reliable-deployments-sw/dist/packages/playground/website/test-fixtures/site-with-unpatched-gutenberg.zip).

I plan to take a look at this after the tests start passing.